library/src/main/java/com/rtchagas/pingplacepicker/PingPlacePicker.kt

```diff
@@ -97,6 +105,7 @@ class PingPlacePicker private constructor() {
     companion object {
 
         const val EXTRA_PLACE = "extra_place"
+        const val ACTUAL_SELECTED_LATLNG = "actual_selected_latlng"
```

Comment:
Please follow the naming standards for extras.
Would be nice if this is named as EXTRA_ACTUAL_LATLNG